src/Command/CheckCommand.php

```diff
@@ -76,19 +76,24 @@ protected function initialize(InputInterface $input, OutputInterface $output): v
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $drupalFinder = new DrupalFinder();
-        $path = realpath($input->getArgument('path'));
 
-        if (!$path || !file_exists($path)) {
-            $output->writeln(sprintf('<error>%s does not exist</error>', $input->getArgument('path')));
-            return 1;
+        $paths = [];
+        foreach ($input->getArgument('path') as $path) {
+            $realPath = realpath($path);
+            if (!$realPath) {
+                $output->writeln(sprintf('<error>%s does not exist</error>', $path));
+                return 1;
+            }
+
+            $paths[] = $realPath;
         }
 
-        $drupalFinder->locateRoot($path);
+        $drupalFinder->locateRoot($paths[0]);
```

Comment:
We should always have at least one element in `$path` at this point, so we'll use the first one to locate the root directory.  (If no paths were given the command would fail during input validation, and if any paths were invalid we would have done a `return 1;` earlier on)